web/styles/left_sidebar.css

```diff
@@ -924,33 +924,33 @@ li.top_left_scheduled_messages {
     grid-template-columns:
         25px $topic_resolve_width minmax(0, 1fr) minmax(0, max-content)
         30px 0;
+}
 
-    .topic-name {
-        cursor: pointer;
-        grid-area: row-content;
-        padding: 1px $before_unread_count_padding 1px 0;
+.topic-box .zero_count {
+    display: none;
+}
 
-        /* TODO: We should figure out how to remove this without changing the spacing */
-        line-height: 1.1;
+.topic-name {
```

Comment:
This is too generic a name for a top-level selector; it's very easy to see random other parts of the app having a `foo class="topic_name"` somewhere.

Reply:
Agreed. I've updated to `sidebar-topic-name`, which aligns better with `inbox-topic-name` and `popover-topic-name`, as well as a structurally adjacent `sidebar-topic-check` class.